File: 9-palindrome-number/palindrome-number.cs
public class Solution {
    public bool IsPalindrome(int x) {
        int n = x;
        int rev =0;
        while(n>0){
            int digi = n%10;
            rev = rev*10+digi;
            n/=10;
        }
        Console.WriteLine(rev);
        if(x != rev){
            return false;
        }
        return true;
    }
}

File: 9-palindrome-number/palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    let r = x;
    let rev = 0;
    if(x<0){
        return false;
    }
    while(r>0)
    {
        let digit = r%10;
        rev = rev*10 + digit;
        r = Math.floor(r/10);
    }
    if(x === rev){
        return true;
    }
    return false;
};